### PR TITLE
SW-4998 Voting page with mock redux/data

### PR DESCRIPTION
**Pull request by karthikbtf**
- TODO: redirect to project management page later
- TODO: show conditional text for conditional votes (follow up with BE changes)

Terraware App 2024-03-12 17-24-39

### CODE REVIEW

**Review by joshbuchea — APPROVED** [review]
Nice work, this looks great!

**Comment by karthikbtf** [comment]
> Nice work, this looks great!

Thanks @joshbuchea

File: src/scenes/AcceleratorRouter/NavBar.tsx [hunk 1]
```diff
@@ -65,7 +68,7 @@ export default function NavBar({ backgroundTransparent, setShowNavBar }: NavBarP
         id='overview'
         label={strings.OVERVIEW}
         onClick={() => closeAndNavigateTo(APP_PATHS.ACCELERATOR_OVERVIEW)}
-        selected={!!isAcceleratorOverviewRoute}
+        selected={!!isOverviewRoute || !!isScoringRoute || !!isVotingRoute || !!isVotingEditRoute}
```
Comment (karthikbtf) [inline src/scenes/AcceleratorRouter/NavBar.tsx:71]: @nickgraz we need this to also include scoring route matches

Comment (nickgraz) [reply src/scenes/AcceleratorRouter/NavBar.tsx:71]: Oh right, I'll add it for now. Nice catch.

**Pull request merged**